api_docs/changelog.md

```diff
@@ -20,6 +20,17 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 8.0
 
+**Feature level 201**
+
+* [`POST /register`](/api/register-queue), [`GET /events`](/api/get-events):
+  `realm_linkifiers` events now send linkifier objects sorted by the order in
+  which they are applied when linkifying messages and topics.
+* [`GET /realm/linkifiers`](/api/get-linkifiers): The returned linkfier objects
+  are now sorted by the order in which they are applied when linkifying messages
+  and topics.
+* [`PATCH /realm/linkifiers`](/api/reorder-linkifiers): Added support to
+  reordering the linkifiers explicitly.
```

Comment:
"Added support for changing the order in which linkifiers will be processed." is better English.